docs/source/specs/v2/openapi.yaml

```diff
@@ -34,6 +34,12 @@ paths:
           schema:
             $ref: '#/components/schemas/Workspaces.WorkspaceTypesQueryParam'
             default: all
+        - name: name
```

Comment:
@astrozzc not sure if it was here, but for v2 we are generating the openapi spec file from typespec: https://github.com/RedHatInsights/insights-rbac/blob/master/docs/source/specs/typespec/main.tsp

Reply:
oh, thanks! i totally forgot about that! Fixed